charts/base-cluster/values.yaml

```diff
@@ -444,3 +451,6 @@ backup:
       limits:
         cpu: 500m
         memory: 500Mi
+
+weaveworks:
+  enabled: false
```

Comment:
Depending on the resource usage we could also deploy it by default, what do you think? @tasches @marvinWolff

Reply:
It does not require a lot of the resources. So we can enabled it per default. Should I leave the enable flag? So the user have the possibility to disable the dashboard if he wants to?